f/export/folder.meta.yaml

```diff
@@ -0,0 +1,2 @@
+summary: A collection of export scripts
```

Comment:
I'm not sure where this shows up. I can't find it back on the Windmill UI, so I might omit.

Also, note the omission of `extra_perms` and `owners`. Syncing works fine without these. So we can remove them from frizzle too (unless there was a good reason to have them).

Reply:
I did not have a good reason at the time.

One question: Once you remove them from frizzle.meta.yaml, does that actually end up changing the permissions in a way we don't want?  I'd imagine it could.

Reply:
>One question: Once you remove them from frizzle.meta.yaml, does that actually end up changing the permissions in a way we don't want? I'd imagine it could.

So these lines get added when you run `wmill sync pull`. When I then remove the lines, and run `wmill sync push`, I see no impact on permissions on the server: I'm able to run, schedule, and edit scripts all the same, schedules are still up, and I see no changes in the UI otherwise. Maybe this is different for a regular admin (not superadmin). But for now I don't see a reason to keep these hard-coded.